quipucords/api/scanjob/tests_scanjob.py

```diff
@@ -215,7 +215,10 @@ def test_pause_restart_task(self, start_scan):
     def test_successful_create(self, start_scan):
         """A valid create request should succeed."""
         data = {'sources': [self.source.id],
-                'scan_type': ScanTask.SCAN_TYPE_CONNECT}
+                'scan_type': ScanTask.SCAN_TYPE_CONNECT,
+                'options': {'disable_optional_products': {'jboss_eap': True,
```

Comment:
I feel like if null had been the default you wouldn't have needed to update so many tests. Just create new ones to test the flow with this value.